.github/workflows/build.yaml

```diff
@@ -6,6 +6,7 @@ on:
   push:
     branches:
       - develop
+      - repo-consistency
```

Comment:
We can probably remove this now right?

Reply:
We'll remove it before we merge, but for the time being we want to confirm that the docker build still succeeds